src/main/java/org/littleshoot/proxy/HttpProxyServerBootstrap.java

```diff
@@ -242,6 +243,14 @@ HttpProxyServerBootstrap withProxyToServerExHandler(
     HttpProxyServerBootstrap withCustomGlobalState(
         GlobalStateHandler globalStateHandler);
 
+    /**
+     *
+     * @param messageProcessorExecutor
+     * @return
+     */
+    HttpProxyServerBootstrap withMessageProcessingExecutor(
+        ExecutorService messageProcessorExecutor);
```

Comment:
will be provided from outside, possibly in a wrapper with added metrics

Reply:
i like the idea of a specialized wrapper.